source/docs/software/roborio-info/roborio2-imaging.rst

```diff
@@ -0,0 +1,56 @@
+RoboRIO2 Imaging
+================
+
+The NI roboRIO 2.0 boots from a microSD card configured with an appropriate boot image containing the NI Linux Real-Time OS, drivers, and libraries specific to FRC. There are two ways to create the SD card -- with the roboRIO using the imaging tool, or with a laptop and an SD burner application.
+
+microSD Requirements
+--------------------
+
+The NI roboRIO 2.0 controller supports microSD cards.
+
+Operation Tips
+--------------
+
+The NI roboRIO 2.0 controller requires a fully inserted SD card containing a valid image in order to boot and operate as intended.
+
+If the SD card is removed while powered, the controller will hang. Once the SD card is replaced, the controller will need to be restarted using the reset button, or be power cycled.
+
+No damage will result from SD card removal or insertion while powered, but best practice is to perform these operations while unpowered.
+
+Imaging Using a Computer
+------------------------
+
+The image will be transferred to the SD card using a specialized writing utility, sometimes called a burner. Several utilities are listed below, but most tools that can write arbitrary images for booting a Raspberry Pi or similar dev boards will also produce a bootable SD card for roboRIO 2.0.
+
+Supported image files are named FRC_roboRIO2_VERSION.img.zip and should be located next to this instruction file. You may also locate then by clicking the SD button in the roboRIO Imaging tool. Select the version that matches the version of the programming tools being used. A mismatch between tools and image may result in inconsistent library dependencies.
+
+If using a non Windows OS you will need to copy this image file to that computer.
```

Comment:
Is this necessary, seems obvious. Also, since you still need the imaging tool to update the team number, it doesn't seem like there's a big use case for programming the SD card on a non-Windows computer.

Reply:
It was asked before where people got the image file on non-windows computers.  I don't know how to judge obviousness beyond that.